app/views/helpers/feed/update.phtml

```diff
@@ -130,6 +130,50 @@
 				</div>
 			</div>
 
+			<div class="form-group">
+				<label class="group-name" for="default_sort"><?= _t('sub.feed.preferred_sort')?> </label>
+				<div class="group-controls">
+					<select name="default_sort_option" id="default_sort_feed" class="w50">
+						<option value="id|DESC" <?=
```

Comment:
We need to add a first option to use the default order (i.e. clear the feed-local preference)